packages/manager/modules/iam/src/components/deleteEntity/deleteEntity.controller.js

```diff
@@ -87,6 +87,8 @@ export default class DeleteEntityController {
       promise = this.deletePolicy();
     } else if (this.entity.type === ENTITY.RESOURCE_GROUP) {
       promise = this.deleteResourceGroup();
+    } else if (this.entity.type === ENTITY.APPLICATION) {
+      promise = this.deleteApplication();
     } else if (this.entity.type === ENTITY.RESOURCE_TYPE) {
       promise = this.$q.when(true);
     } else {
```

Comment:
Maybe begin to replace by catch to readability and anticipate evolution ?

Reply:
switch/case sorry

Reply:
Could be fixed in a next PR when it's merged